pages/index.js

```diff
@@ -41,12 +46,15 @@ export async function getServerSideProps({ req, res, query }) {
         initialImage = selectRandomImage(HAVE_STREAM_IMAGE_SET)
     }
 
+    pastStream = await pollPaststreamStatus(process.env.WATCH_CHANNEL_ID)
```

Comment:
Following from my other comment on pollPaststreamStatus, we will need to do the same unpacking dance as pollLivestreamStatus here.